app/types/strings/Ability.ts

```diff
@@ -610,6 +610,12 @@ export const AbilityName: { [key in Ability]: Langage } = {
     prt: ``,
     fra: ``
   },
+  [Ability.JUDGEMENT]: {
```

Comment:
I split many mythical abilities into their own unique abilities so that it no longer relies on the "stars" property, because this property was also used for the CSV export to identify the expected strength of this mon, and therefore at which stage it should be proposed